src/ocamldep.ml

```diff
@@ -129,61 +129,57 @@ let deps_of cctx ~ml_kind ~already_used unit =
     match Module.file ~dir unit ml_kind with
     | None -> Build.return []
     | Some file ->
-      let all_deps_path file =
-        Path.extend_basename file ~suffix:".all-deps"
+      let file_in_obj_dir ?(suffix="") file =
```

Comment:
Do you have plans to use this function with passing `~suffix`? Otherwise, it's simpler to just get rid of it and also use the chance to avoid the `suffix=""` optimization.

Reply:
I agree.

Reply:
indeed, i didn't notice it's always set